### PR TITLE
Add applied modules count in header

### LINKED ISSUE #3989: Display applied modules count in header
We can display the number of applied modules in header using the modules statistics

### PR DESCRIPTION
**Pull request by nassimerrahoui**
Close #3989

### CODE REVIEW

**Review by DamnClin — APPROVED** [review]

File: src/test/javascript/spec/common/secondary/RestStatisticsRepository.spec.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { RestStatistics } from '@/common/secondary/RestStatistics';
+import { RestStatisticsRepository } from '@/common/secondary/RestStatisticsRepository';
+import { dataBackendResponse, stubAxiosHttp } from '../../http/AxiosHttpStub';
+
+describe('Rest statistics repository', () => {
+  it('Should get statistcs using axios', async () => {
```
Comment (DamnClin) [inline src/test/javascript/spec/common/secondary/RestStatisticsRepository.spec.ts:6]: statistcs -> statistics

File: src/test/javascript/spec/common/domain/Statistics.spec.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Statistics } from '@/common/domain/Statistics';
+
+describe('Statistcs', () => {
+  it('Should not be negative', () => {
```
Comment (DamnClin) [inline src/test/javascript/spec/common/domain/Statistics.spec.ts:4]: 'Should not be negative' -> 'Should not build with negative application count'

File: src/test/javascript/spec/common/domain/Statistics.spec.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { Statistics } from '@/common/domain/Statistics';
+
+describe('Statistcs', () => {
```
Comment (DamnClin) [inline src/test/javascript/spec/common/domain/Statistics.spec.ts:3]: Statistcs -> Statistics

**Review by DamnClin — APPROVED** [review]

File: src/test/javascript/spec/common/secondary/RestStatisticsRepository.spec.ts (hunk 1 above)
Comment (nassimerrahoui) [reply src/test/javascript/spec/common/secondary/RestStatisticsRepository.spec.ts]: Done 👍

File: src/test/javascript/spec/common/domain/Statistics.spec.ts (hunk 2 above)
Comment (nassimerrahoui) [reply src/test/javascript/spec/common/domain/Statistics.spec.ts]: Done 👍

File: src/test/javascript/spec/common/domain/Statistics.spec.ts (hunk 3 above)
Comment (nassimerrahoui) [reply src/test/javascript/spec/common/domain/Statistics.spec.ts]: Done 👍

**Comment by pascalgrimaud** [comment]
Merging this as it's better to have this, than nothing :)
But there are still this comment: https://github.com/jhipster/jhipster-lite/issues/3989#issuecomment-1286925721

**Pull request merged**